pkg/diagnostics/networkpod/util/util.go

```diff
@@ -31,6 +31,8 @@ const (
 	NetworkDiagNodeLogDirPrefix      = "/nodes"
 	NetworkDiagMasterLogDirPrefix    = "/master"
 	NetworkDiagPodLogDirPrefix       = "/pods"
+
+	NetworkDiagDefaultPodImage = "openshift/origin"
```

Comment:
Any reason this should not use [`variable.DefaultImagePrefix`](https://github.com/openshift/origin/blob/master/pkg/cmd/util/variable/imagetemplate.go#L24-L26) ?